NEWS.md

```diff
@@ -1,3 +1,83 @@
+# Sparklyr 1.3.0
+
+### Spark ML
+
+- `ml_evaluate()` methods are implemented for ML clustering and classification models
+
+### Distributed R
+
+- Created helper methods to integrate Spark SQL higher-order functions with
+  `dplyr::mutate`
+
+- Implemented option to pass partition index as a named parameter to `spark_apply()`
+  transform function
+
+- Enabled transform function of `spark_apply()` to return nested lists
+
+- Added option to return R objects instead of Spark data frame rows from transform
+  function of `spark_apply`
+
+- `sdf_collect()` now supports fetching Spark data frame row-by-row rather than
+  column-by-column, and fetching rows using iterator instead of collecting all
+  rows into memory
+
+### Connections
+
+- Hive integration can now be disabled by configuration in `spark_connect()` (#2465)
+
+- A JVM object reference counting bug affecting secondary Spark connections was fixed
+  (#2515)
+
+- Revised JObj envs initialization for Databricks connections (#2533)
+
+### Serialization
+
+- The new `arrow::write_to_raw()` functionality will be used if available in arrow-
```

Comment:
This isn't really interesting to a user of `sparklyr`. It's essentially a refactoring of the `arrow` code with no change in functionality.